File: NG-Training/NG.NET/TaskTrackerApp/src/app/interfaces/task.ts
export interface Task {
    id: string;
    userId: string;
    clientName: string;
    projectName: string;
    taskTitle: string;
    hours: number;
    dateTime: Date;
    assignedTo: string;
    assignedBy: string;
    taskState: string;
    priority: string;
    description: string;
}

File: .NET-Training/Backend-Training/StandardApiSolution/KnowledgeBase/Models/Task.cs
namespace KnowledgeBaseApi.Models
{
    public class Task
    {
        public Guid? Id { get; set; }
        public Guid UserId { get; set; }
        public required string ClientName { get; set; }
        public required string ProjectName { get; set; }
        public required string TaskTitle { get; set; }
        public decimal Hours { get; set; }
        public DateTime DateTime { get; set; }
        public required string AssignedTo { get; set; }
        public required string AssignedBy { get; set; }
        public required string TaskState { get; set; }
        public required string Priority { get; set; }
        public required string Description { get; set; }
    }
}
